packages/cli/src/UserManagement/UserManagementHelper.ts

```diff
@@ -223,9 +218,12 @@ export function isAuthenticatedRequest(request: express.Request): request is Aut
 //            hashing
 // ----------------------------------
 
-export async function compareHash(str: string, hash: string): Promise<boolean | undefined> {
+export const hashPassword = (validPassword: string): string =>
+	hashSync(validPassword, genSaltSync(10));
+
+export async function compareHash(plaintext: string, hash: string): Promise<boolean | undefined> {
 	try {
-		return await compare(str, hash);
+		return await compare(plaintext, hash);
```

Comment:
This is correct because it is in a try-catch, so we have to await the promise to make sure it did not error. In fact our ruleset currently [requires us to await](https://github.com/typescript-eslint/typescript-eslint/blob/v4.33.0/packages/eslint-plugin/docs/rules/return-await.md#in-try-catch).

<img width="1002" alt="image" src="https://user-images.githubusercontent.com/44588767/161523834-c9b4aa29-3893-4c2d-b534-de11bf72a337.png">